modules/dnn/test/test_model.cpp

```diff
@@ -299,7 +305,43 @@ TEST_P(Test_Model, Keypoints_face)
 
     Mat inp = imread(_tf("gray_face.png"), 0);
     std::string weights = _tf("onnx/models/facial_keypoints.onnx", false);
-    Mat exp = blobFromNPY(_tf("facial_keypoints_exp.npy"));
+    float kpdata[] = {
+       -1.1784188f , -0.28653762f,       -1.1314853f ,  0.01879984f,
+       -1.0556153f ,  0.3161617f ,       -0.9805776f ,  0.59290093f,
+       -0.8553336f ,  0.9090538f ,       -0.6502977f ,  1.219669f  ,
+       -0.3969034f ,  1.4355791f ,       -0.08717892f,  1.6473793f ,
+        0.32542017f,  1.7758257f ,        0.6898433f ,  1.6232896f ,
+        0.90219647f,  1.4329152f ,        1.0506458f ,  1.2165918f ,
+        1.2101227f ,  0.96450716f,        1.3160325f ,  0.6647914f ,
+        1.3689535f ,  0.40930197f,        1.4283009f ,  0.15772615f,
+        1.4500887f , -0.15895039f,       -0.7646912f , -0.4223808f ,
+       -0.58158964f, -0.50166416f,       -0.37167928f, -0.48618358f,
+       -0.17388777f, -0.42147297f,        0.00546351f, -0.3428334f ,
+        0.71651113f, -0.35376707f,        0.90102017f, -0.40692362f,
+        1.0235933f , -0.47311983f,        1.1844229f , -0.48181295f,
+        1.319576f  , -0.3905492f ,        0.34878936f,  0.02133076f,
+        0.4036541f ,  0.24864897f,        0.4169562f ,  0.49212712f,
+        0.4217249f ,  0.64887923f,        0.14697891f,  0.65257084f,
+        0.23538592f,  0.69651544f,        0.3831995f ,  0.71456987f,
+        0.47956857f,  0.7063694f ,        0.56533986f,  0.6379722f ,
+       -0.50764734f, -0.08413627f,       -0.34318796f, -0.08306785f,
+       -0.18520775f, -0.07939741f,       -0.03210878f, -0.03949469f,
+       -0.15310487f,  0.02115617f,       -0.3536016f ,  0.01545431f,
+        0.68299335f, -0.03518461f,        0.84433997f, -0.09107257f,
+        0.9807276f , -0.10665712f,        1.0828875f , -0.08523301f,
+        0.9618248f ,  0.03302464f,        0.82244664f,  0.00855381f,
+       -0.13033527f,  0.9488608f ,        0.02154717f,  0.8673497f ,
+        0.25307602f,  0.81715864f,        0.36301583f,  0.8185618f ,
+        0.4858782f ,  0.8166637f ,        0.64733803f,  0.8720045f ,
+        0.75894463f,  0.96210575f,        0.6322932f ,  1.1599172f ,
+        0.5222149f ,  1.282458f  ,        0.3378732f ,  1.2986604f ,
+        0.19809961f,  1.2634348f ,        0.02682695f,  1.1636689f ,
+       -0.11073969f,  0.9403825f ,        0.22035715f,  0.9148466f ,
+        0.38391942f,  0.9214617f ,        0.4866761f ,  0.93515944f,
+        0.7235508f ,  0.9525098f ,        0.49401033f,  1.1250464f ,
+        0.3664117f ,  1.1356674f ,        0.18867175f,  1.0959036f
+    };
+    Mat exp(68, 2, CV_32FC1, kpdata);
```

Comment:
Let's keep `facial_keypoints_exp.npy`

Reply:
ok, i'll do that.

should i change to the new model, containing concatenated heat&paf maps, and add some code to the model like:

```
        // slice output, if it's the lightweight_human_pose one, or the openpose model with original prototxt
        if (nPoints == 57)
        {
            nPoints = 19;
        }
```

or keep it as is (model w. 19 heatmaps, code unchanged) ?

Reply:
I think it's better to keep as is for now. It's easier for user to process output.

Reply:
ok, done, thanks for the quick reply.